server/src/main/java/org/elasticsearch/index/query/LegacyGeoShapeQueryProcessor.java

```diff
@@ -63,6 +63,11 @@ public LegacyGeoShapeQueryProcessor(AbstractGeometryFieldMapper.AbstractGeometry
         this.ft = ft;
     }
 
+    @Override
+    public Query process(Geometry shape, String fieldName, ShapeRelation relation, QueryShardContext context) {
+        return process(shape, fieldName, LegacyGeoShapeFieldMapper.DeprecatedParameters.Defaults.STRATEGY, relation, context);
```

Comment:
I think it might be safer to throw an Unsupported Operation exception here to make sure we don't call this accidentally.

Reply:
:+1: